frontend/src/customAxios.ts

```diff
@@ -69,13 +66,8 @@ const CustomAxios = ({
         lifecycleToasts.errorToast();
       }
 
-      const status = error.response ? error.response.status : null;
       //const errorMessage =
       //  errorToastMessage || (error.response && error.response.data.message) || String.ERROR;
-
-      if (status === UNAUTHORIZED || status === MAINTENANCE) {
-        window.location.reload(false);
```

Comment:
I think this is the most likely cause of your issue @Fosol . This logic could easily get into a loop where it would try and load a url, get a 401, reload, and then get a 401 again indefinitely. This inspired me to find and remove the other non-react router navigations throughout the app.